core/src/main/java/tc/oc/pgm/filters/modifier/QueryModifier.java

```diff
@@ -10,6 +11,8 @@
  * Takes a query and modifies it before passing it onto its child filter. Might transform query type
  * so {@link #getQueryType()} on this modifier can be different from its child filter
  *
+ * <p>If the modified query returns null the query will abstain.
```

Comment:
```suggestion
 * <p>If the modified query returns {@code null} this modifier will abstain without querying the child filter
```

A query can't abstain, unsure about the phrasing.